src/extlibs/hrvo/path_point.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+#include <vector>
+
+#include "software/geom/vector.h"
+/**
+ * A path point in an agent's path
+ */
+class PathPoint
+{
+   public:
+    /**
+     * @param position  The position of this path point
+     */
+    explicit PathPoint(const Vector &position);
```

Comment:
To make the value of `speed_at_position` more explicit to the user, maybe we should remove this constructor and have the user only use the other one. What do you think?

Reply:
Yeah that makes sense, I'll make the change